sdcm/nemesis.py

```diff
@@ -580,31 +581,69 @@ def disrupt_nodetool_refresh(self, big_sstable=True, skip_download=False):
             #        In my old test, the instance init is easy to fail. We can try to use a
             #        split shared disk to save the 100GB file.
             # 100M (500000 rows)
-            sstable_url = 'https://s3.amazonaws.com/scylla-qa-team/refresh_nemesis/keyspace1.standard1.100M.tar.gz'
+            sstable_url = 'https://s3.amazonaws.com/scylla-qa-team/refresh_nemesis_c0/keyspace1.standard1.100M.tar.gz'
             sstable_file = '/tmp/keyspace1.standard1.100M.tar.gz'
             sstable_md5 = '9c5dd19cfc78052323995198b0817270'
+            keys_num = 501000
         else:
-            sstable_url = 'https://s3.amazonaws.com/scylla-qa-team/refresh_nemesis/keyspace1.standard1.tar.gz'
+            sstable_url = 'https://s3.amazonaws.com/scylla-qa-team/refresh_nemesis_c0/keyspace1.standard1.tar.gz'
             sstable_file = "/tmp/keyspace1.standard1.tar.gz"
             sstable_md5 = 'c033a3649a1aec3ba9b81c446c6eecfd'
-        if not skip_download:
-            key_store = KeyStore()
-            creds = key_store.get_scylladb_upload_credentials()
-            remote_get_file(self.target_node.remoter, sstable_url, sstable_file,
-                            hash_expected=sstable_md5, retries=2,
-                            user_agent=creds['user_agent'])
+            keys_num = 1000
```

Comment:
I don't understand why we still have this part.

Reply:
Currently I used IF judgement to choose the right snapshot with expected columns number.
I can make it simple

Reply:
But on the bottom part you have the exact same code and the exact same locations of the sstables.

Reply:
Just fixed the typo.
I'm testing with 3 byo jobs (1 column, 5 columns, 10 columns).